### PR TITLE
[ISSUE #302]fea(doc) RocketMQ Streams doc

### PR DESCRIPTION
**Pull request by ni-ze**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by ni-ze** [comment]
close https://github.com/apache/rocketmq-site/issues/302

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# RocketMQ Streams 概览
+RocketMQ Streams是基于RocketMQ的轻量级流计算引擎。他能以SDK方式被应用依赖，无须部署复杂的流计算服务端即可获得流计算能力。
```
Comment (chenzlalvin) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md:2]: 文档描述建议不要用拟人第三人称描述组件

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+计算实例实质上是依赖了Rocket-streams SDK的client，因此，计算实例消费的MQ依赖RocketMQ rebalance分配，
+计算实例总个数也不能大于消费总MQ个数，否则将有部分计算实例处于等待状态，消费不到数据。
+
+一个计算实例可以消费多个MQ，一个实例内也只有一张计算拓扑图。
+
+## 状态
+![img_3.png](../picture/33rocketmq-streams/状态存储.png)
+
+对于有状态算子，比如count，需要先对count算子进行分组，然后才能求和。分组算子groupBy会将数据按照分组的key重新写回RocketMQ，并且使相同key写入同一分区（这一过程称作shuffle），
+保证这个含有相同key的数据被同一个消费者消费。 他的状态本地依赖RocksDB加速读取，远程依赖RocketMQ做持久化，在做checkpoint时将本地RocksDB中状态写入到RocketMQ中。
```
Comment (chenzlalvin) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md:26]: 同上

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# RocketMQ Streams 概览
```
Comment (chenzlalvin) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md:1]: 建议补充一个章节 RocketMQ Streams的典型应用场景，做一下方案简单的优势对比。

Comment (ni-ze) [reply i18n/en/docusaurus-plugin-content-docs/version-5.0/08-流计算/30RocketMQ Streams Overview.md:1]: 列了一个典型应用场景

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**